appmonitoring/scripts/validate_ai.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+POD_DOTNET_NAME=$1
+POD_JAVA_NAME=$2
+POD_NODEJS_NAME=$3
+AI_RES_ID=$4
+
+# Get an access token
+result_rsp=$(curl 'http://169.254.169.254/metadata/identity/oauth2/token?api-version=2018-02-01&resource=https://api.applicationinsights.io&mi_res_id=/subscriptions/66010356-d8a5-42d3-8593-6aaa3aeb1c11/resourceGroups/rambhatt-rnd-v2/providers/Microsoft.ManagedIdentity/userAssignedIdentities/rambhatt-agentpool-es-identity' -H Metadata:true -s)
+# echo "Result: $result_rsp"
+access_token=$(echo $result_rsp | jq -r '.access_token')
+
+echo $AI_RES_ID
+
+# Define your variables
+url="https://api.loganalytics.io/v1$AI_RES_ID/query"
+
+verify_AI_telemetry() {
+    echo $1
+    json_body="{
+        'query': 'traces | where timestamp > ago(5m) | where cloud_RoleName == \"$1\" | count',
```

Comment:
5 minutes is questionable, will break if there's delay in Kusto. Can we make this 15 minutes?

Reply:
done